reinforcement_learning/custom-recipes/Choose the environment, the agent and train it/recipe.json

```diff
@@ -0,0 +1,256 @@
+// This file is the descriptor for the Custom code recipe Choose the environment, the agent and train it
+{
+    // Meta data for display purposes
+    "meta" : {
+        "label" : "Training",
+        "description" : "Choose the environment, the agent and train it",
+        "icon" : "icon-gamepad"
+    },
+
+    "kind" : "PYTHON",
+    
+
+    // Inputs and outputs are defined by roles. In the recipe's I/O tab, the user can associate one
+    // or more dataset to each input and output role.
+
+    // The "arity" field indicates whether the user can associate several datasets to the role ('NARY')
+    // or at most one ('UNARY'). The "required" field indicates whether the user is allowed to
+    // associate no dataset with the role.
+
+"outputRoles" : [
+    {
+        "name": "main_output",
+        "label": "Saved Models",
+        "description": "The folder that will contain the saved models",
+        "arity": "UNARY",
+        "required": true,
+        "acceptsDataset": false,
+        "acceptsManagedFolder":true
+    }
+],
+
+"params": [ 
```

Comment:
These are very specific parameters. Don't write too much but I'm sure there is a way to have short desc that explain what the thing does and/or why to change it.